Gemfile

```diff
@@ -25,7 +25,10 @@ gem 'link_header'
 
 # Publish Admin
 gem 'devise'
-gem 'administrate', github: 'thoughtbot/administrate'
+gem 'activeadmin'
+gem 'cancan'
```

Comment:
Do we need these additional gems? If so, is there a reason they aren't dependencies of active admin?

Reply:
I think they can probably go until we actually need them.